src/components/ChainSelectionPopover/ChainSelectionPopover.styles.tsx

```diff
@@ -0,0 +1,92 @@
+import styled from "styled-components/macro";
+
+import breakPoints from "../../style/breakpoints";
+import { BorderlessButtonStyle } from "../../style/mixins";
+
+/**
+ * @remarks props are used to manipulate the positioning of components
+ */
+type ContainerProps = {
+  open: boolean;
+  shiftLeft: boolean;
+};
+
+export const Container = styled.div<ContainerProps>`
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  width: 16rem;
+  height: 14rem;
```

Comment:
```suggestion
  height: auto;
```

This can not be fixed, what if we want to have 5 buttons?

Reply:
I removed `height` and replaced it with `min-height: 100%;`, which not sizes it dynamically